drivers/watchdog/wdt_mcux_wwdt.c

```diff
@@ -82,7 +82,8 @@ static int mcux_wwdt_install_timeout(const struct device *dev,
 		return -ENOMEM;
 	}
 
-#if defined(CONFIG_SOC_MIMXRT685S_CM33) || defined(CONFIG_SOC_MIMXRT595S_CM33)
+#if defined(CONFIG_SOC_MIMXRT685S_CM33) || defined(CONFIG_SOC_MIMXRT595S_CM33) \
+	|| defined(CONFIG_SOC_SERIES_MCXNX4X)
```

Comment:
just put this in the same commit as the dts entry, a separate commit is overkill, both are basically one line changes to achieve the same goal